build/bin/upload

```diff
@@ -26,12 +26,17 @@ then
 fi
 
 echo "Archiving files from $must_gather_output to $must_gather_upload/${CSP_FILENAME}"
-tar caf "$must_gather_upload/${CSP_FILENAME}" $must_gather_output/
+tar --ignore-failed-read -caf "$must_gather_upload/${CSP_FILENAME}" $must_gather_output/
 
 echo "Uploading '${CSP_FILENAME}' to Red Hat Customer SFTP Server for case ${caseid}"
 
 REMOTE_FILENAME=${caseid}_${CSP_FILENAME}
 
+if [[ "${internal_user}" == true ]]; then
+  # internal users must upload to a different path on the sftp
+  REMOTE_FILENAME="${username}/${REMOTE_FILENAME}"
+fi
+
```

Comment:
@dustman9000 here we decided to implement some logic based on whether the MGO user is an internal hydra user or not. most SREP folks are not internal hydra users (unless they came from CEE, like myself), while all of CEE should be marked as internal users in hydra. the team who manages the SFTP server apparently changed the way uploads work for the SFTP server back in February 2021. with these changes, internal users can now only upload files to their home directories on the SFTP server, while users who are not  marked internal can only upload to the / dir on the server. this effectively broke MGO for our CEE users, as the must-gathers were not getting uploaded to the SFTP server (I'm not sure why nobody has mentioned it since February, though, and I'm sad about what that implies)